demisto_sdk/commands/common/tools.py

```diff
@@ -1587,6 +1588,12 @@ def find_type_by_path(path: Union[str, Path] = "") -> Optional[FileType]:
         elif PARSING_RULES_DIR in path.parts:
             return FileType.PARSING_RULE
 
+        elif MODELING_RULES_DIR in path.parts:
+            return FileType.MODELING_RULE
+
+        elif CORRELATION_RULES_DIR in path.parts:
+            return FileType.CORRELATION_RULE
```

Comment:
Strange to have it twice, here and here: https://github.com/demisto/demisto-sdk/blob/fbebca74c9c15efa9a4b497b08101e899d40f24d/demisto_sdk/commands/common/tools.py#L1674
Why did you add it? It did not work without it?